src/modules/game.js

```diff
@@ -0,0 +1,13 @@
+export default async function setNewGame(name) {
+  const API_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+  const request = new Request(API_URL, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ name: `${name}` }),
+  });
+  const response = await fetch(request);
+  return response.json();
+}
```

Comment:
- [ ] Please change this function to the arrow function. You can check this. [why you should use the arrow function.](https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/)